images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -14,4 +14,86 @@ Describe "Kind" {
     It "Kind" {
         "kind version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Jq" {
+    It "Jq" {
+        "jq -n ." | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Julia" {
+    It "Julia path exists" {
+        "C:\Julia" | Should -Exist
```

Comment:
just to clarify: why do we need to check Julia path? I mean we are not checking path for other tools in this file, just its version, but we do it for Julia

Reply:
We manually set up installation path as a parameter in the `Install-Julia.ps1` script "--ia","/DIR=C:\Julia"

Reply:
got it, thanks!